core/arch/atomic_exports.h

```diff
@@ -961,11 +964,11 @@ atomic_compare_exchange_int64(volatile int64 *var, int64 compare, int64 exchange
      * sequentially consistent memory order semantics.
```

Comment:
We only need acquire/release: we don't need sequential consistency.

Reply:
Fixed in https://github.com/DynamoRIO/dynamorio/pull/6089/commits/43eaccfa161c78355d61b04bbcd317e3466edebd.